### PR TITLE
Setup themes

### PR DESCRIPTION
**Pull request by ivankristianto**
- Add SVG icons
- Add link to github, facebook group and slack team
- styling

### CODE REVIEW

File: themes/wpid/functions.php [hunk 1]
```diff
@@ -16,3 +16,17 @@ function wpid_enqueue_assets() {
 }
 add_action( 'wp_enqueue_scripts', 'wpid_enqueue_assets' );
 
+
+/**
+ * Add SVG definitions to the footer.
+ */
+function wpid_include_svg_icons() {
+	// Define SVG sprite file.
+	$svg_icons = get_parent_theme_file_path( '/assets/images/svg-icons.svg' );
```
Comment (kucrut) [inline themes/wpid/functions.php]: Nice, didn't know about `get_parent_theme_file_path()`. :+1:

File: themes/wpid/style.css [hunk 2]
```diff
@@ -23,13 +23,73 @@ body {
...
+	margin-left: 0;
+	padding-left: 0;
+}
+
+.social-navigation li {
+	display: inline;
+}
+
+.social-navigation a {
+	background-color: #767676;
```
Comment (kucrut) [inline themes/wpid/style.css]: The gray background colour for the icons doesn't look really good here. Can we use another colour? `transparent` would be OK.

File: themes/wpid/style.css [hunk 3]
```diff
@@ -23,13 +23,73 @@ body {
...
+	-webkit-border-radius: 40px;
+	border-radius: 40px;
+	color: #fff;
+	display: inline-block;
+	height: 60px;
+	margin: 0 1em 0.5em 0;
```
Comment (kucrut) [inline themes/wpid/style.css]: The right margin is causing the icons to not aligned in center. Maybe change this to `margin: 0 .5em 0.5em;`?

File: themes/wpid/style.css (hunk 2 above)
Comment (ivankristianto) [inline themes/wpid/style.css]: `transparent` not really good. i change it to black for now.

**Pull request merged**